client/src/helper-functions/rnRHelper.js

```diff
@@ -107,15 +107,16 @@ const helper = {
     });
   },
   filterReviewsByStars: (reviews, starFilters) => {
-
     if (starFilters.length === 0) {
       return reviews
     }
     return reviews.filter((review) => {
       return starFilters.includes(review.rating);
     })
   },
-
+  auditReviews: (review) => {
```

Comment:
looks like your removing extra spaces -  just wanted to say i dig how clean your code is always looking and the effort you take to clean it up 🔥